spec/lite_spec_helper.rb

```diff
@@ -82,13 +82,13 @@ def example_timeout_seconds
     config.add_formatter(RSpec::Core::Formatters::JsonFormatter, File.join(File.dirname(__FILE__), '../tmp/rspec.json'))
   end
 
-  if SpecConfig.instance.ci? && !%w(1 true yes).include?(ENV['INTERACTIVE']&.downcase)
-    config.around(:each) do |example|
-      TimeoutInterrupt.timeout(example_timeout_seconds) do
-        example.run
-      end
-    end
-  end
+  # if SpecConfig.instance.ci? && !%w(1 true yes).include?(ENV['INTERACTIVE']&.downcase)
+  #   config.around(:each) do |example|
+  #     TimeoutInterrupt.timeout(example_timeout_seconds) do
+  #       example.run
+  #     end
+  #   end
+  # end
```

Comment:
Are these lines commented out because they might come back? Should we just delete them and rely on git to remember them if we need to resurrect them?

Reply:
Oh, good catch! This should be removed eventually, I believe, but not in scope of this PR. I'll revert.